gameplay/avalonRoom.js

```diff
@@ -1629,14 +1643,16 @@ module.exports = function (host_, roomId_, io_, maxNumPlayers_, newRoomPassword_
 			if (this.host === socket.request.user.username) {
 				
 				this.socketsOfPlayers.splice(getIndexFromUsername(this.socketsOfPlayers, username), 1);
-				
-				console.log("Kicked player: " + username);
-
-				this.sendText(this.socketsOfPlayers, "Player " + username + " has been kicked by the host.", "server-text");
-
 				// Ban them from this room
-				this.kickedPlayers[username] = true;
-
+				// Add ban time start
+				this.kickedPlayers[username] = {
+					"status": true,
```

Comment:
Is "status" ever false? If its always true, do we need it?

Reply:
Good point. Was about ban times.

Accepted. Will be addressed in the next commit.